libraries/encrypted-db/build.gradle.kts

```diff
@@ -34,4 +34,6 @@ dependencies {
     implementation(libs.sqlcipher)
     implementation(libs.sqlite)
     implementation(libs.androidx.security.crypto)
+
+    implementation(projects.libraries.androidutils)
```

Comment:
Maybe, but we want to avoid having unused declared dependencies, because it will increase the build time if the dependency is modified. Ideally the CI would detected unused dependency so we could do some cleanup.